src/ceph_fuse.cc

```diff
@@ -161,6 +161,7 @@ int main(int argc, const char **argv, const char *envp[]) {
       return r;
     }
     if (forker.is_parent()) {
+      signal(SIGHUP, SIG_IGN);
```

Comment:
Instead of fixing it here we'd better fix it in the forker.prefork(), not only the ceph-fuse have the problem.

Reply:
> Instead of fixing it here we'd better fix it in the forker.prefork(), not only the ceph-fuse have the problem.

Thank you for your review and reply, I have modified it.